manualTests/build.gradle.kts

```diff
@@ -187,9 +188,9 @@ xtcRun {
      * TODO: Add a nicer DSL syntax with a nested modules section.
      */
```

Comment:
You are doing something with the launcher in this branch, while I struggle to finish the build system. Can you please only modify the launcher code in this branch, and not do unrelated changes. I'm review ready in 30 minutes, and the manual tests do not look like that anymore.

Reply:
If you feel it is an important change that TestFizzBuzz changes names to EchoTest, please still do it in a separate small PR, so that it does not widen the impact area of conflict resolution.